Makefile

```diff
@@ -203,14 +203,19 @@ add-test-user:
 ##     are ready.
 ##     In case of timeout, the application will be redeployed ONLY if TEST is set
 wait-until-ready:
-	@printf "\nWaiting for readiness of platform"; \
+	@{ printf "\nWaiting for readiness of platform"; \
+	DOCKER_TAG=$(DOCKER_TAG); \
+	if [ "$${DOCKER_TAG}" = "latest" ]; then \
+	  export DOCKER_TAG=test; \
+	fi; \
```

Comment:
Does not this break local development with the default latest tag?

Reply:
wait until ready is not dependent on DOCKER_TAG. DOCKER_TAG is only important when the redeployment kicks-in.  redeployment is only allowed when the TEST variable is set. So when TEST variable is set - it is assumed that if no tag is set, it is "test". If no tag is set, the DOCKER_TAG will be set to "latest" automatically. Therefore the check. If you provide any other tag, it should work (if TEST is set). But you will probably never test with "latest" tag.

Reply:
Ok makes sense. Approving.